binary_search/BinarySearch.java

```diff
@@ -0,0 +1,34 @@
+public class BinarySearch {
+
+    static int binarySearch(int[] arr, int arrSize, int searchElement) {
```

Comment:
Javadoc comment for function means need to some comments in main function. Is that saying?